### PR TITLE
Integrate projector

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
Needs https://github.com/OpenSlides/openslides-autoupdate-service/pull/169

### CODE REVIEW

File: client/src/app/slides/base-slide-component.ts [hunk 1]
```diff
@@ -13,7 +13,9 @@ export abstract class BaseSlideComponent<T extends object> {
      * Each slide must take slide data.
      */
     @Input()
-    public set data(value: SlideData<T>) {}
+    public set data(value: SlideData<T>) {
+        this._data = value;
+    }
 
     public get data(): SlideData<T> {
         return this._data;
```
Comment (GabrielInTheWorld) [inline client/src/app/slides/base-slide-component.ts:21]: Could this be unified to `@Input() public data: SlideData<T>`?

**Review by GabrielInTheWorld — COMMENTED** [review]
Looks good. One little Remark.

File: client/src/app/slides/base-slide-component.ts (hunk 1 above)
Comment (FinnStutzenstein) [reply client/src/app/slides/base-slide-component.ts:21]: Yes, right. It seems to be a leftover from debugging.

I'll fix this when implementing all other slides.

Comment (FinnStutzenstein) [reply client/src/app/slides/base-slide-component.ts]: nevermind, fixed.

Comment (FinnStutzenstein) [reply client/src/app/slides/base-slide-component.ts]: ah, no. It is currently needed by some child components.

**Pull request merged**